addons/mail/tests/test_mail_activity.py

```diff
@@ -162,3 +162,23 @@ def test_mail_activity_schedule_from_chatter(self):
             "mail_activity_schedule_from_chatter",
             login="admin",
         )
+
+    def test_mail_activity_date_format(self):
+        with freeze_time("2024-1-1 00:00:00"):
+            LANG_CODE = "en_US"
+            self.env = self.env(context={"lang": LANG_CODE})
```

Comment:
I would avoid patching test datetime at midnight, just because this is at the very point of transition between 2 days and this can be confusing and/or prone to issues. `10:00` or something is good for example